### PR TITLE
Exposed 'requiresName' props, and allows option to remove CardHolderName field

### PR DESCRIPTION
**Pull request by tons613**
Exposed 'requiresName' props, and allows option to remove CardHolderName field

### CODE REVIEW

File: src/components/Card/FrontSide.tsx [hunk 1]
```diff
@@ -19,13 +19,11 @@ type Props = {
...
-  const { overrides, translations } = useContext(LibraryContext)
+  const { overrides, translations, requiresName } = useContext(LibraryContext)
   const [numberLayout, setNumberLayout] = useState<LayoutRectangle | null>(null)
   const [nameLayout, setNameLayout] = useState<LayoutRectangle | null>(null)
-  const [
-    expirationLayout,
-    setExpirationLayout,
-  ] = useState<LayoutRectangle | null>(null)
+  const [expirationLayout, setExpirationLayout] =
+    useState<LayoutRectangle | null>(null)
```
Comment (halilb) [inline src/components/Card/FrontSide.tsx]: This change looks like not modified on purpose.

**Review by halilb — COMMENTED** [review]
Looks good. Thanks for the contribution @tons613.

Could you check my comment and also add `requireName` prop to the README file documentation? I'll merge the PR afterwards.

File: src/components/Card/FrontSide.tsx (hunk 1 above)
Comment (tons613) [reply src/components/Card/FrontSide.tsx]: Yes, sorry it was not modified on purpose. I guess my editor code prettier did. I'm fixing it now

Comment (tons613) [reply src/components/Card/FrontSide.tsx]: Done! @halilb

**Review by halilb — APPROVED** [review]
Looks good! Thank you @tons613.

**Pull request merged**